dist_sidouglas/static/css/app.7a8a71755a6da1afbf9cd7e45c5e5506.css

```diff
@@ -0,0 +1 @@
+#app{font-family:Avenir,Helvetica,Arial,sans-serif;-webkit-font-smoothing:antialiased;-moz-osx-font-smoothing:grayscale;text-align:center;color:#2c3e50;margin-top:60px}.inline-style{display:-ms-flexbox;display:flex;border:1px solid blue}.inline-style:after{outline:1px solid green;content:"this works only when it's inline."}h1,h2{font-weight:400}ul{list-style-type:none;padding:0}li{display:inline-block;margin:0 10px}a{color:#42b983}.hello{display:flex;flex-flow:column;outline:1px solid red}.hello:after{content:"( Expect to see -ms-flex and -ms-flex-flow vendor styles in css output for ie10 for the .hello div )"}
```

Comment:
.hello element is missing `-ms-flex` rules for `display:flex` and `flex-flow`. See .hello:after rule for expectation.